gradlew

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env sh
 
-# added manually
-./config/hooks/install-pre-commit
```

Comment:
Isn't this the GJF hook installer? Why are we getting rid of it?

Reply:
@lazaroclapp I just opened #297 on this.  We should find a different way to do this.  `gradlew` is an auto-generated file so this was a hack all along.

Reply:
I vote we re-add this for now, then we remove it again at the same time as we implement #297

Reply:
Sure, but then let's land #298 before this one. We really shouldn't let master be red with respect to this, hack or not :)